napari/components/viewer_model.py

```diff
@@ -973,18 +981,10 @@ def add_vectors(
         return layer
 
     def _new_points(self):
```

Comment:
can we just drop the `_new_points` method entirely and connect the button to `self.add_points()` instead - one less function to worry about. similar for `_new_shapes` too

Reply:
Yes but I worried about existing tests and having to replicate/modify them. It's almost 1am here. ;) I'd prefer to leave that for a later PR.